src/Features/Core/Portable/LanguageServices/SymbolDisplayService/AbstractSymbolDisplayService.AbstractSymbolDescriptionBuilder.cs

```diff
@@ -151,6 +151,41 @@ protected SemanticModel GetSemanticModel(SyntaxTree tree)
 
                 AddOverloadCountPart(symbols);
                 FixAllAnonymousTypes(symbols[0]);
+                AddExceptions(symbols[0]);
+            }
+
+            private void AddExceptions(ISymbol symbol)
+            {
+                var exceptionTypes = symbol.GetDocumentationComment().ExceptionTypes;
+                if (exceptionTypes.Any())
+                {
+                    var parts = new List<SymbolDisplayPart>();
+                    parts.Add(new SymbolDisplayPart(kind: SymbolDisplayPartKind.Text, symbol: null, text: $"\r\n{WorkspacesResources.Exceptions}"));
+                    foreach (var exceptionString in exceptionTypes)
+                    {
+                        parts.AddRange(LineBreak());
+                        parts.AddRange(Space(count: 2));
+
+                        // try to get the actual exception symbol
+                        var exceptionSymbol = DocumentationCommentId.GetFirstSymbolForDeclarationId(exceptionString, _semanticModel.Compilation);
+                        if (exceptionSymbol != null)
+                        {
+                            parts.AddRange(_displayService.ToMinimalDisplayParts(_semanticModel, _position, exceptionSymbol));
+                        }
+                        else
+                        {
+                            // unable to parse exception symbol, fall back to displaying the raw text after trying to
```

Comment:
@jasonmalinowski: I found where crefs were getting their "T:" prefix stripped and exposed the method as `internal` so it could be re-used here.  See latest commit.